src/events/EventsPageBanner.tsx

```diff
@@ -49,20 +50,13 @@ export const EventsPageBanner: React.FC<EventsPageBannerProps> = ({
             <del>Coming Out</del>
             <ins>Staying In</ins>
           </Title>
-          <Subtitle>
+          <P marginBottom={'1.5rem'} color={'white'}>
```

Comment:
- `marginBottom` as this takes the `styled-system: space` prop, we can use the spacing system here, ie just pass the relative spacing through. This could for example be `sm` to indicate it should be "small" on each screen size (the spacing scale is defined in the `src/theme/space.ts` file and should shed more light on what each value is per screen size. This means that eg `xl` would be smaller on a mobile screen than it would be on a desktop screen, but it would maintain the ratio.

- `color`: this should come from the `src/theme/colors.ts` module 🙂